.circleci/config.yml

```diff
@@ -182,6 +182,16 @@ workflows:
                 - "quay.io/pypa/manylinux_2_24_x86_64" # PEP 600
               python-tags:
                 - "cp38-cp38" # Python 3.8
+      - build-wheel:
+          name: "build-wheel-cp310-manylinux_2_24"
+          requires:
+            - build-wheel-cp38-manylinux_2_24
+          matrix:
+            parameters:
+              platform:
+                - "quay.io/pypa/manylinux_2_24_x86_64" # PEP 600
+              python-tags:
+                - "cp310-cp310" # Python 3.10
```

Comment:
You can also remove this whole thing by simply adding cp310-cp310 to the prior section's matrix. But the name would need to be changed from build-wheel-cp38-manylinux_2_24 to build-wheel-<<matrix.python-tags>>-manylinux_2_24